internal/pkg/container/config.go

```diff
@@ -27,3 +27,9 @@ func ImageParentDir() string {
 func ImageFile(name string) string {
 	return path.Join(ImageParentDir(), name+".img")
 }
+
+// InitramfsBootPath returns the dracut built initramfs path, as dracut built initramfs inside container
+// the function returns host path of the built file
+func InitramfsBootPath(image, kver string) string {
+	return path.Join(RootFsDir(image), "boot", "initramfs-"+kver+".img")
```

Comment:
Can you make sure that Suse and Debian both use this same path and naming scheme for dracut images? If not, we should make sure we can detect those, too.